server/src/internalClusterTest/java/org/opensearch/remotemigration/RemoteStoreMigrationTestCase.java

```diff
@@ -42,6 +43,11 @@ protected int minimumNumberOfReplicas() {
         return 1;
     }
 
+    @Override
+    protected Settings featureFlagSettings() {
+        return Settings.builder().put(super.featureFlagSettings()).put(FeatureFlags.REMOTE_STORE_MIGRATION_EXPERIMENTAL, "true").build();
```

Comment:
Why do we need this override? As I understand, the feature flag now controls Remote Store to DocRep migration.

Reply:
`testMigrationDirections` tests `docrep` direction as well .